cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/queue/hbase/ShardedHBaseQueueStrategy.java

```diff
@@ -68,7 +69,7 @@
   // Number of bytes as the row key prefix, including salt bytes added by the row key distributor
   public static final int PREFIX_BYTES = SaltedHBaseQueueStrategy.SALT_BYTES + Bytes.SIZEOF_LONG + Bytes.SIZEOF_INT;
 
-  private static final Function<byte[], byte[]> ROW_KEY_CONVERTER = new Function<byte[], byte[]>() {
+  static final Function<byte[], byte[]> ROW_KEY_CONVERTER = new Function<byte[], byte[]>() {
```

Comment:
E.g. you can have something like `createHBaseScanner` for returning the `ResultScanner` instead of `QueueScanner` and have only that method package protected, but not other field/method in this class.